src/components/Map/styles.scss

```diff
@@ -8,6 +8,7 @@
 
   &--desktop {
     flex-direction: row;
+    height: 700px;
```

Comment:
Is this overridable? Wondering if some instances might need a larger or smaller height